defi.tokenlist.json

```
@@ -8,10 +8,10 @@
     ],
     "version": {
         "major": 2,
-        "minor": 20,
+        "minor": 21,
         "patch": 0
     },
-    "timestamp": "2021-08-23T21:19:53.238+00:00",
+    "timestamp": "2021-08-24T21:19:53.238+00:00",
```

Timestamp can't be in the future.

```suggestion
    "timestamp": "2021-08-24T17:00:00.000+00:00",
```